frontend/packages/console-shared/src/components/status/icons.tsx

```diff
@@ -7,23 +7,22 @@ import {
 import {
   global_warning_color_100 as warningColor,
   global_danger_color_100 as dangerColor,
+  chart_color_green_400 as okColor,
```

Comment:
Is there a global_success_color_100 instead of chart_color_green_400? I'm not sure what the values are for these but using a chart variable for an icon could cause issues. There is a var(--pf-global--success-color--100) available but I'm not sure if these are 1-1.

Reply:
Good catch Matt! `var(--pf-global--success-color--200)` looks best here, this should probably be fixed Console-wide.

Reply:
This comes from #2930, I will fix there